### PR TITLE
sf-2333: add support for transfer folders endpoint

### PR DESCRIPTION
**Pull request by rklind**

### CODE REVIEW

File: src/commands/jobs/util.ts [hunk 1]
```diff
@@ -263,9 +263,9 @@ export const itemOutputFormat = {
 				if (!val) {
 					return '';
 				}
-				const size = !Boolean(val.source_to_destination)
-					? (val.source && val.source.bytes)
-					: (val.destination && val.destination.bytes);
+				const size = Boolean(val.source_to_destination) && val.destination && val.destination.bytes
+					? (val.destination && val.destination.bytes)
+					: (val.source && val.source.bytes);
```
Comment (rklind) [inline src/commands/jobs/util.ts]: does this change make sense or should i create a different output format for folder size? my concern with this logic is that if an item has 0 bytes on the destination, but it's not 0 bytes on the source, you'd see the source size, which may give a false impression of the contents on the destination...if there was a problem with the transfer.

File: src/commands/jobs/util.ts [hunk 2]
```diff
@@ -263,9 +263,9 @@ export const itemOutputFormat = {
 				if (!val) {
 					return '';
 				}
-				const size = !Boolean(val.source_to_destination)
-					? (val.source && val.source.bytes)
-					: (val.destination && val.destination.bytes);
+				const size = Boolean(val.source_to_destination) && val.destination && val.destination.bytes
```
Comment (smbecker) [inline src/commands/jobs/util.ts]: I think that this statement should be included within the `Boolean` expression.

File: src/sdk/resources/transferFolders.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	}
+	
+	async list(params?: any): Promise<TransferItem[]> {
+		this.resourcePath = `transfers/${params.id}/folders`;
+		delete params.id;
+		return super.list(params);
+	}
+}
+
+// const result = await this.httpClient.get(`transfers/${id}/folders`, this.mergeDefaultParams(params));
```
Comment (smbecker) [inline src/sdk/resources/transferFolders.ts]: ??

Comment (rklind) [reply src/sdk/resources/transferFolders.ts]: oops, thanks!

File: src/commands/jobs/util.ts (hunk 2 above)
Comment (rklind) [reply src/commands/jobs/util.ts]: will do!

**Pull request merged**